modal/volume.py

```diff
@@ -390,8 +390,6 @@ async def read_file(self, path: Union[str, bytes]) -> AsyncIterator[bytes]:
         print(len(data))  # == 1024 * 1024
         ```
         """
-        if isinstance(path, str):
-            path = path.encode("utf-8")
```

Comment:
how does it work now – will protobuf automatically run the same things when you set the field value?